### PR TITLE
Feat added single video

### PR DESCRIPTION
**Pull request by raman1999**
image

### CODE REVIEW

File: src/Components/index.js [hunk 1]
```diff
@@ -2,3 +2,6 @@ export { Navbar } from "./Navigation/Navigation";
 export { Footer } from "./Footer/Footer";
 export { Sidebar } from "./Sidebar/Sidebar";
 export { VideoGrid } from "./VideoGrid/VideoGrid";
+export { AddToPlaylistContainer } from "./AddToPlaylistContainer/AddToPlaylistContainer";
+export { Sidebar } from "./Sidebar/Sidebar";
```
Comment (rahulcs754) [inline src/Components/index.js]: you can set name inside sidebar index js

File: src/Pages/SingleVideo/SingleVideo.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { useParams } from "react-router-dom";
+import { useUserContext } from "../../Context";
+import "./single_video.css";
+import { updateHistory } from "../../Services";
+import ReactPlayer from "react-player";
+import { AddToPlaylistContainer } from "../../Components";
+import { useState } from "react";
+import { UseGetAxios } from "../../Hooks/UseGetAxios";
+import { LoadingSpinner } from "../../Components";
```
Comment (rahulcs754) [inline src/Pages/SingleVideo/SingleVideo.jsx]: no need line number 9 you can do in line number 6

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good 
Small suggestion

File: src/Components/AddToPlaylistContainer/AddToPlaylistContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import { PlaylistModal } from "../index";
+import "./playlist_container.css";
+
+export function AddToPlaylistContainer({ videoDetails }) {
+  const {
+    userState: { playlists },
+    userDispatch,
+  } = useUserContext();
+  const [showModal, setShowModal] = useState(false);
+  console.log(videoDetails);
```
Comment (sadathshariff) [inline src/Components/AddToPlaylistContainer/AddToPlaylistContainer.jsx]: remove console.log

**Review by sadathshariff — APPROVED** [review]
Looks Good and clean code

File: src/Pages/SingleVideo/SingleVideo.jsx (hunk 2 above)
Comment (raman1999) [reply src/Pages/SingleVideo/SingleVideo.jsx]: Ok

File: src/Components/AddToPlaylistContainer/AddToPlaylistContainer.jsx (hunk 3 above)
Comment (raman1999) [reply src/Components/AddToPlaylistContainer/AddToPlaylistContainer.jsx]: i will remove it

**Pull request merged**